config/routes.rb

```diff
@@ -1,5 +1,13 @@
 Rails.application.routes.draw do
-  devise_for :users
+  get :dashboard, to: 'companies#dashboard'
```

Comment:
use member and collection group routes functionality instead of writing each single route

Reply:
Added Temporary